### PR TITLE
REST - Remove unneeded dependent jar files

### PR DESCRIPTION
**Pull request by venkat1m**
This is address jira TRAFODION-1829

Cleaned up the pom.xml and the assembly package all.xml and reduced the dependencies to the absolute minimum.

httpcore-4.1.2.jar	( Apache 2.0 )
jettison-1.3.5.jar	( Apache 2.0 )
jetty-http-9.2.10.v20150310.jar	( Apache 2.0 )
jetty-io-9.2.10.v20150310.jar	( Apache 2.0 )
jetty-server-9.2.10.v20150310.jar	( Apache 2.0 )
jetty-servlet-9.2.10.v20150310.jar ( Apache 2.0 )
jetty-util-9.2.10.v20150310.jar	( Apache 2.0 )
jython-standalone-2.5.3.jar	( Apache 2.0 )
zookeeper-3.4.5.jar	( Apache 2.0 )
slf4j-api-1.7.5.jar	( MIT Licensed )
slf4j-log4j12-1.7.5.jar	( MIT Licensed )
javax.servlet-api-3.1.0.jar	( CDDL 1.0 )
jersey-json-1.8.jar	( CDDL1.1 )
jersey-server-1.8.jar	( CDDL1.1 )

### CODE REVIEW

File: core/rest/pom.xml [hunk 1]
```diff
@@ -417,29 +406,19 @@
   	<compileSource>1.6</compileSource>
     
   	<!-- Dependencies -->
-    <hadoop.version>${env.HADOOP_DEP_VER}</hadoop.version>
+    <hadoop.version>2.6.0</hadoop.version>
```
Comment (DaveBirdsall) [inline core/rest/pom.xml]: Do we want to hard-code this? If my memory serves, this was parameterized only recently.

**Comment by DaveBirdsall** [comment]
Looks good to me. One question though.

**Comment by venkat1m** [comment]
Remove the hard-coded hadoop version and revert back to using the environment variable. This was a merge issue.

**Comment by DaveBirdsall** [comment]
Thanks for fixing that. Looks good to me.

File: core/rest/pom.xml [hunk 2]
```diff
@@ -599,73 +523,6 @@
     <!-- While jackson is also a dependency of both jersey and avro, these
          can bring in jars from different, incompatible versions. We force
          the same version with these dependencies -->
```
Comment (arvind-narain) [inline core/rest/pom.xml]: looks good to me Venkat. Should these comments related to jackson be removed too ?

**Comment by DaveBirdsall** [comment]
Is this ready for merging? Or will there be a further change?

**Comment by venkat1m** [comment]
Yes. this is ready for merging. Please commit. I will address Arvinds comment in a separate pull request, as it is only about removing some comments in pom.xml

**Comment by DaveBirdsall** [comment]
It looks like you were able to delete the comments after all... When the tests complete, I'll commit this change.

**Pull request merged**